### DISCUSSION TITLE
Durée des formulaires sur l'environnement QA

### DISCUSSION THREAD

**Discussion by MpmMess** [MTESSDev/FRW#14] [Annonces]
Nous avons remarqué qu'après un certain temps les formulaires FRW déjà instanciés sont supprimés (environnement QA). Nous aimerions valider avec vous, si ce comportement est normal ? C'est surtout lors de nos essais pilotes que ceci nous impacté. 

Quel est le délai du temps avant que le formulaire soit supprimé de l'env. QA ?

Pouvons-nous faire augmenter ce délai par configuration ?

**Comment by loima02** [comment]
Salut MpmMtess!

L'épuration des formulaires dans l'environnement QA est présentement quotidienne. J'avoue que c'est un peu intense pour vos besoins.

À ce jour, je ne peux pas la configurer "par système autorisé", mais je vais augmenter le délai d'épuration.

Combien de jours ferait votre affaire?

**Reply by MpmMess** [reply]
Salut @loima02, 

Voici la réponse qui a envoyé notre pilote : Idéalement 10 jours, sinon 5 pour laisser le temps au responsable de produit et au pilote de système d’effectuer leurs essais d’acceptations à chaque fin de sprint. 

Merci,

**Reply by loima02** [reply]
Approuvé pour 10 jours! Ce sera effectif dans l'après-midi.

**Reply by MpmMess** [reply]
Merci, je communique l'information à mes collègues. 

Bonne journée!
